hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -293,8 +293,10 @@ def _get_item_info(self, path):
             (category_type, _) = mimetypes.guess_type(filename)
 
             if category_type:
-                # the category portion of the mimetype
-                category = category_type.split("/")[0]
+                # the category portion of the mimetype. needs to be encoded to utf-8 since category members are used in constructing items,
```

Comment:
suggest: limit the length of comment lines to `80` characters for consistency with the rest of this code.